Quadriga/src/main/resources/locale/messages_en_US.properties

```diff
@@ -136,9 +136,15 @@ username.required=Username is required.
 name.required=We'd like to know your name.
 password.required=Please specify a password.
 password.notmatch=Passwords don't match.
-
+
 # account approval
 account_approval.error=Sorry, the account couldn't be approved. There seems to be a general problem with account approval.
 account_approval.account_roles.required=Please specify at least one role for the user.
 account_approval.action.required=Please select "Approve" or "Deny".
-
+
+# Text Management
+filename.required = Filename is required.
+filecontent.required = File content is required.
+filename.proper = The filename should only contain the following characters: 0-9, a-Z, _, and -.
+reference.required = A reference is required.

```

Comment:
pleaes add the following to this message: "Please provide the URI of the original copy of the text."